app/assets/stylesheets/alerts.scss

```diff
@@ -0,0 +1,12 @@
+.alert-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+#error_explanation {
```

Comment:
Avoid using id selectors<br>Selector `error_explanation` should be written in lowercase with hyphens